Gemfile.lock

```diff
@@ -261,6 +264,8 @@ GEM
     fauxhai-ng (9.3.0)
       net-ssh
     ffi (1.17.2)
+    ffi (1.17.2-arm64-darwin)
+    ffi (1.17.2-x64-mingw-ucrt)
```

Comment:
I guess this is not supposed to be here, has nothing to do with `chef-zero`?

Reply:
but we do need them both. Adding them wasn't my doing, they just popped in when updating chef-zero

Reply:
ok, taking it out then.